atlp-rwanda/atlp-devpulse-fn
# #149 Resend verification email

### PR Description
This PR is for the user to choose resend verification email after the account is created.

### Description of tasks that were expected to be completed
-  Creating the email templlates to be sent to the emails inbox

### How has this been tested?
- Clone repo
- Checkout branch `ft-email-verification`
- Make sure that you set the environment variables referring to `.env.example`
- Run `npm install`, then `npm run dev` to check if the app is running successfully
- Then you will create account to see the outputs

### Vercel preview
[Link](https://atlp-devpulse-fn-git-ft-emai-16b358-devpulses-projects-62c56cf1.vercel.app/)

### Track PR (issue number & link)
#149 

### Screenshots
![Screenshot 2024-09-29 112111](https://github.com/user-attachments/assets/f46e7aaa-3947-4e42-86f6-213813b635fa)
